src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
```

Comment:
The image should be an `img` tag instead of a background-image for better accessibility. Consider adding an `img` tag in the HTML and applying styles to it.